manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -316,26 +344,26 @@ func (r *FybrikApplicationReconciler) reconcile(applicationContext *api.FybrikAp
 		if err := r.deleteExternalResources(applicationContext); err != nil {
 			return ctrl.Result{}, err
 		}
-		r.Log.V(0).Info("no blueprint will be generated since no datasets are specified")
+		r.Log.Info().Str(logging.FYBRIKAPPUUID, uuid).Msg("No plotter will be generated since no datasets are specified")
 		return ctrl.Result{}, nil
 	}
 
 	// create a list of requirements for creating a data flow (actions, interface to app, data format) per a single data set
 	// workload cluster is common for all datasets in the given application
 	workloadCluster, err := r.GetWorkloadCluster(applicationContext)
 	if err != nil {
-		r.Log.V(0).Info("could not determine in which cluster the workload runs: " + err.Error())
+		r.Log.Fatal().Err(err).Bool(logging.FORUSER, true).Bool(logging.AUDIT, true).Str(logging.FYBRIKAPPUUID, uuid).Str(logging.ACTION, logging.CREATE).Msg("Could not determine in which cluster the workload runs")
 		return ctrl.Result{}, err
 	}
 	var requirements []DataInfo
 	for _, dataset := range applicationContext.Spec.Data {
 		req := DataInfo{
 			Context: dataset.DeepCopy(),
 		}
-		r.Log.V(0).Info("Preparing requirements for " + req.Context.DataSetID)
+		r.Log.Trace().Str(logging.FYBRIKAPPUUID, uuid).Str(logging.DATASETID, req.Context.DataSetID).Str(logging.ACTION, logging.CREATE).Msg("Preparing requirements for dataset")
```

Comment:
It is not a create action. All controller functionality is about constructing a Plotter, does not seem right to have a create action for all printouts.

Reply:
I guess we need to come up with guidelines about when to use CREATE, UPDATE, DELETE in the log messages.  Do you have thoughts about this?

Reply:
My thought was that the action is a k8s action applied to a resource or resources, such as CREATE, UPDATE, DELETE, GET, LIST. This is an important information for admin, developers, audit - each one for a different reason.

Reply:
We could do that I guess.  Inside the reconcile functions though we often don't know if we are creating or updating though.  Even the k8s call is createorupdate.